BLOGGING.md

```diff
@@ -0,0 +1,141 @@
+# Writing and Publishing a Pulumi Blog Post
+
+So you're interested in contributing to the Pulumi blog? Great! Follow these steps to make it happen.
+
+## Set Up Your Development Environment
+
+If you haven't already, clone this repository and [follow the instructions in the README](https://github.com/pulumi/docs#pulumi-documentation-site) to set up your environment and run the development web  server. Once you're able to run
+
+```
+make serve
+```
+
+and browse the site locally, you're ready to move on to the next section.
+
+## Make a New Post
+
+1. Resist the temptation to copy-and-tweak an existing post! Instead, paste the following command into the terminal (at the root of the project) to generate a new one:
+
+    ```
+    hugo new --kind blog-post blog/my-new-post
+    ```
+
+    This will create a minimal post browseable at http://localhost:1313/blog/. You'll find the new post's source file at `content/blog/my-new-post/_index.md` containing the set of [Hugo front matter](https://gohugo.io/content-management/front-matter/) properties you'll need to get started:
```

Comment:
```suggestion
    This will create a minimal post browsable at http://localhost:1313/blog/. You'll find the new post's source file at `content/blog/my-new-post/_index.md` containing the set of [Hugo front matter](https://gohugo.io/content-management/front-matter/) properties you'll need to get started:
```